include/networkit/GlobalState.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef NETWORKIT_GLOBAL_STATE_HPP_
+#define NETWORKIT_GLOBAL_STATE_HPP_
+
+#ifndef NETWORKIT_EXPORT
+#if defined(WIN32)
+#define NETWORKIT_EXPORT __declspec(dllexport)
+#else
+#define NETWORKIT_EXPORT
+#endif
+#endif // NETWORKIT_EXPORT
```

Comment:
This looks broken. Surely we also need `dllimport`? We should set an appropriate variable in the CMake script to distinguish between export and import (= the `NETWORKIT_BUILDING_STATELIB` that I mentioned earlier).

Reply:
Also, there is still no indentation.

Reply:
For intendation we need to change `.clang_format` config. This will be done in a seperate PR, since it likely changes more than only a few files.